lints.rs

```diff
@@ -13,36 +13,45 @@
     clippy::exit,
     clippy::explicit_into_iter_loop,
     clippy::filter_map_next,
+    clippy::float_cmp_const,
     clippy::fn_params_excessive_bools,
+    clippy::get_unwrap,
     clippy::if_let_mutex,
     clippy::imprecise_flops,
     clippy::inefficient_to_string,
     clippy::let_unit_value,
     clippy::linkedlist,
     clippy::lossy_float_literal,
     clippy::macro_use_imports,
+    clippy::map_err_ignore,
     clippy::map_flatten,
     clippy::map_unwrap_or,
     clippy::match_on_vec_items,
     clippy::match_wildcard_for_single_variants,
     clippy::mem_forget,
     clippy::mismatched_target_os,
+    clippy::mutex_integer,
```

Comment:
`mutex_integer` lint has not triggered in our codebases, but it is a simple one and been on the our low-priority list to enable

Reply:
Seems quite unlikely for this to end up in a codebase, and is usually caught during manual review anyway unless there are very specific requirements that cannot be modeled with a simple atomic.